src/parser.rs

```diff
@@ -178,6 +185,22 @@ pub fn parse(pairs: Pairs<Rule>) -> Result<Items, Error<Rule>> {
                         body,
                     })
                 }
+                Rule::table => {
+                    let mut items = parse(pair.into_inner())?.into_iter();
```

Comment:
This is fine, but a small rust-point: I think we can use something like https://github.com/max-sixty/prql/blob/906b951ab1be39e6fda9df69bf163d700c216674/src/parser.rs#L126 to make it more concise, WDYT?

But it's also OK to merge and we can come back to refine the rust if you prefer